test/test_umart.py

```diff
@@ -0,0 +1,39 @@
+"""
+Tests for umart.py
+"""
+from test.conftest import MockUQCSBot, TEST_CHANNEL_ID
+from unittest.mock import patch
+import codecs
+
+NO_QUERY_MESSAGE = "You can't look for nothing. `!umart <QUERY>`"
+NO_RESULTS_MESSAGE = "I can't find anything. Try `!umart <SOMETHING NOT AS SPECIFIC>`"
+ERROR_MESSAGE = "I tried to get the things but alas I could not. Error with HTTP Request."
+
+TEST_URL = "https://www.umart.com.au/umart1/pro/products_list_searchnew_min.phtml?search=HDD&bid=2"
+
+
+def get_test_message():
```

Comment:
pls no :(
I think you can just inline this as the expected response for now

Reply:
Do you mean to move the message into the test_umart file?

Reply:
just have some shit like:

```
GOOD_RESPONSE = """Name: <https://www.umart.com.au/umart1/pro/https://www.umart.com.au/umart1/pro/Products-details.phtml?id=10&bid=2&id2=55&sid=365951|Panasonic Toughbook CF-54 (14.0") Mk3 500GB HDD, 8GB Performance>
Price: $1999.00
Name: <https://www.umart.com.au/umart1/pro/https://www.umart.com.au/umart1/pro/Products-details.phtml?id=10&bid=2&id2=66&sid=360422|Toshiba PS591A-03L011  C50 i5-8250U 15.5" HD 8GB 1TB HDD, WL  DVDRW W10P>
Price: $1099.00
Name: <https://www.umart.com.au/umart1/pro/https://www.umart.com.au/umart1/pro/Products-details.phtml?id=10&bid=2&id2=67&sid=353479|Acer AN515-51-77DM i7-7700H 15.6"FHD LCD(1920x1080) NV1050Ti/4GB16GB 128GBSSD+1TBHDD HDMI,HD Cam SD>
Price: $1399.00
Name: <https://www.umart.com.au/umart1/pro/https://www.umart.com.au/umart1/pro/Products-details.phtml?id=10&bid=2&id2=67&sid=354463|Acer Aspire A515-51G-72DK i7-7500U/15.6" FHD/NVIDIA Geforce MX150-2GB/4GB+8GB Ram/128GB SSD+1TB HDD>
Price: $1269.00
Name: <https://www.umart.com.au/umart1/pro/https://www.umart.com.au/umart1/pro/Products-details.phtml?id=10&bid=2&id2=67&sid=358004|Acer Aspire A517-51G-70WJ  i7-7500U 17.3" FHD/940MX-2GB/4GB+8GB/128GB SSD+1TB HDD/DVDSM/W10H>
Price: $1239.00
"""
```
at the top, don't need it as a text file (html we probably should since its super long)

Reply:
also try to cut this example down as much as possible ^^